Documentation/Login/Index.rst

```diff
@@ -0,0 +1,26 @@
+.. include:: ../Includes.txt
+
+.. _login:
+
+=====
+Login
+=====
+
+In TYPO3 CMS an editor's work is done in the backend. So point your browser
```

Comment:
"So point your browser ..." is duplicate to line 17-19. I suggest to remove this sentence here.